.circleci/config.yml

```diff
@@ -95,27 +95,39 @@ workflows:
 
   build:
     jobs:
-      - checkout
+      - checkout:
+          context: 
+          - Globality-Common
```

Comment:
What is `Globality-Common`?

Reply:
It is where we consolidate all the environment variables